packages/vscode-dependencies-validation/src/commands.ts

```diff
@@ -0,0 +1,40 @@
+import { dirname } from "path";
+import type { DiagnosticCollection } from "vscode";
+import { invokeNPMCommand } from "@sap-devx/npm-dependencies-validation";
+import { refreshDiagnostics } from "./diagnostics";
+import { VscodeOutputChannel } from "./vscodeTypes";
+
+export async function fixAllDepIssuesCommand(
+  outputChannel: VscodeOutputChannel,
+  packageJsonPath: string,
+  dependencyIssuesDiagnosticCollection: DiagnosticCollection
+): Promise<void> {
+  outputChannel.show(true);
+
+  const npmCommand = "install";
+  try {
+    outputChannel.appendLine(`\nFixing dependency issues ...`);
+    const start = Date.now();
+    const config = { commandArgs: [npmCommand], cwd: dirname(packageJsonPath) };
+    await invokeNPMCommand(config, outputChannel);
+
+    outputChannel.append(`Done. (${Date.now() - start} millis)\n`);
```

Comment:
I think for using facing output we would prefer to count in seconds

Reply:
Done